### DISCUSSION TITLE
Is there an ARM64 version of ejabberd?

### DISCUSSION THREAD

**Discussion by ughstudios** [processone/ejabberd#3771] [Ideas]
https://www.process-one.net/en/ejabberd/downloads/

It looks like there is no arm-64 package? Could this be made? I can save a lot of money running this on AWS Graviton servers. However, those only run on ARM-64 bit version of Ubuntu/Linux distros.

**Comment by badlop** [comment]
Hi, there was a recent issue dealing with ARM packages, see https://github.com/processone/ejabberd/issues/3762
In practice, there are:
* a repository by [ballerburg9005](https://github.com/ballerburg9005), see https://github.com/processone/docker-ejabberd/issues/71
* a recent PR by [psjbeisler](https://github.com/psjbeisler), see * https://github.com/processone/docker-ejabberd/pull/79

If you try any (or both) of those solutions, please provide feedback (good or bad): comment if they work, if you had to fix something for them to work, if you are happy with them and work great...

**Comment by licaon-kter** [comment]
@ughstudios you can always build from source, it's a short process :)

**Comment by badlop** [comment]
Since [ejabberd 22.05](https://github.com/processone/ejabberd/releases/tag/22.05) there are binary installers for linux/arm64, and there's also a [container image for arm64](https://github.com/processone/ejabberd/pkgs/container/ejabberd).
